apidoc/Titanium/App/iOS/UserActivity.yml

```diff
@@ -159,6 +162,12 @@ methods:
         Marks the activity as currently **not** in use and ineligible to be continued.
     platforms: [iphone, ipad]
     osver: {ios: {min: "9.0"}}
+  - name: isSupported
+    summary: Indicates whether userActivity is supported by the device.
+    returns:
+            type: Boolean
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "8.0"}}
```

Comment:
- Use the description from the old method instead of a new descriptin
- Fix `[App.iOS.UserActivity.getSupported()](App.iOS.UserActivity.isSupported())` with `[Ti.App.iOS.UserActivity.isSupported()](Titanium.App.iOS.UserActivity.isSupported)`
- Make sure the `notes` are not too long and break the sentence with `|` into two lines.
- The `type` attribute has a wrong spacing.